Formula/apophenia.rb

```diff
@@ -0,0 +1,31 @@
+class Apophenia < Formula
+  desc "C library for statistical and scientific computing"
+  homepage "http://apophenia.info/"
+  url "https://github.com/b-k/apophenia/archive/v1.0.tar.gz"
+  sha256 "c753047a9230f9d9e105541f671c4961dc7998f4402972424e591404f33b82ca"
+
+  depends_on "gsl"
+  depends_on "sqlite"
```

Comment:
Unless this needs a particularly recent sqlite, you can omit this requires_on in order to use the OS X-supplied sqlite3 instead.